package.json

```diff
@@ -28,6 +28,7 @@
     "test:all": "jest --runInBand",
     "test:app": "jest --runInBand ./App",
     "test": "jest --runInBand __tests__",
+    "install:all": "cd ./release/app && npm install && cd ../.. && npm install",
```

Comment:
nit: can we update the "Install development and runtime node dependencies:" section to use this instead of `npm install` please?